app/assess/models/flag.py

```diff
@@ -8,18 +8,14 @@ class FlagType(Enum):
     FLAGGED = 0
     STOPPED = 1
     QA_COMPLETED = 2
-
-
-class ResolutionType(Enum):
-    QUERY_RESOLVED = 0
-    STOP_ASSESSMENT = 1
+    RESOLVED = 3
 
 
 @dataclass()
 class Flag:
+    id: str
```

Comment:
could this be a UUID type?

Reply:
So this is a pattern across all the dataclasses on the frontend. Its a uuid on the [datamodel class](https://github.com/communitiesuk/funding-service-design-assessment-store/pull/98/files#:~:text=%3D%20%22flags%22-,id%20%3D%20db.Column(,),-justification%20%3D%20db) in the database but on the frontend we just pass it around as a string until it returns. back to the store (we dont actually use the flag id yet anyway)